### ISSUE TITLE
System font not fully replaced in Finder

### ISSUE THREAD

**Issue by rcpolich** [jenskutilek/FiraSystemFontReplacement#51]
I followed the instructions, and Fira seems to populate everywhere except for file and folder names using Column View in Finder (which is ironically where San Francisco bothers me most). Screenshot attached; items in the sidebar use Fira, and items in Icon or List View do as well. Just not Column View. I'm running Sierra 10.12.6.

![screen shot 2017-12-14 at 9 01 26 am](https://user-images.githubusercontent.com/34549680/34004770-a597060e-e0ad-11e7-9d4f-d563d704c105.png)
![screen shot 2017-12-14 at 9 02 25 am](https://user-images.githubusercontent.com/34549680/34004771-a6061a9e-e0ad-11e7-826b-d28c877aad4e.png)
![screen shot 2017-12-14 at 9 02 27 am](https://user-images.githubusercontent.com/34549680/34004772-a616b48a-e0ad-11e7-9958-5d4ac9df4381.png)

**Comment by jenskutilek** [comment]
Good catch ... it seems that the column view in Finder uses the Display version of the system font. That makes no sense ... The Display version is not included in the replacement fonts yet. I don't know when I will find the time to add it.

**Comment by rcpolich** [comment]
That is really strange. Thanks for clarifying what's going on. Thanks as
well for your hard work in making Fira Sans a possibility! Every time I
update my OS, it's the first change I make—it's such a dramatic and
important improvement over Apple's choice of fonts. Keep up the good work!

**Comment by xujc** [comment]
There is no such problem on my Mac (`MacBook Pro (Retina, 13-inch, Late 2013), macOS High Sierra Version 10.13.2`). @rcpolich 

<img width="497" alt="screen shot 2017-12-16 at 08 52 48" src="https://user-images.githubusercontent.com/7146271/34065637-d49e567c-e23e-11e7-8447-e944d37bf131.png">

By the way, thank @jenskutilek  very much for making the Fira font replacement which looks much better than the default font.
